packages/maskbook/src/plugins/Transak/index.ts

```diff
@@ -0,0 +1,11 @@
+import { registerPlugin } from '@masknet/plugin-infra'
+import { base } from './base'
+
+registerPlugin({
+    ...base,
+    SNSAdaptor: {
```

Comment:
It needs a dashboard entry.

Reply:
Do you intend to reply for previous conversation above?

Reply:
If it is going to be opened by useRemoteControlledDialog in the dashboard, you should add it.

Reply:
Both `Transak` and `Swap` will be opened by `useRemoteControlledDialog` in dashboard 2.0

Reply:
Fixed in https://github.com/DimensionDev/Maskbook/pull/3496/commits/f193092eec774ff15b9f6ce025e82c0f637e8e3b Thanks